x-pack/plugins/reporting/server/config/schema.ts

```diff
@@ -9,16 +9,7 @@ import { ByteSizeValue, schema, TypeOf } from '@kbn/config-schema';
 import moment from 'moment';
 
 const KibanaServerSchema = schema.object({
-  hostname: schema.maybe(
-    schema.string({
-      validate(value) {
-        if (value === '0') {
-          return 'must not be "0" for the headless browser to correctly resolve the host';
-        }
-      },
-      hostname: true,
-    })
-  ),
+  hostname: schema.maybe(schema.string({ hostname: true })),
```

Comment:
@elastic/kibana-reporting-services `0` is now correctly considered as an invalid hostname by `joi`. We're loosing the custom error message as the internal validation failing is performed before calling it.

Can you confirm this is acceptable?